File: mario-more/mario.c
#include<stdio.h>
#include<cs50.h>
void pyramid(int h)
{
    for(int i = 1; i <= h; i++)
    {
        for(int j = h; j > 0; j--)
        {
            if(i>=j)
            {
                printf("#");
            }
            else
            {
            printf(" ");
            }
        }
        printf("  ");
        for(int j = 1; j <= h; j++)
        {
            if(i>=j)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");
    }
}

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }while((height <= 0) || (height > 8));
    pyramid(height);
    return 0;
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while (True):
    height = get_int("Height: ")
    if(height > 0 and height <= 8):
        break
    else:
        print("Height should be a positive integer but not greater than 8.")

for i in range(height):
    for j in range(height-1, i-1, -1):
        if (i == j):
            print("#" * (i+1), end="")
        else:
            print(" ", end="")

    print("  ", end="")

    for k in range(height):
        if (i == k):
            print("#" * (i+1), end="")
            continue
    print()